dev-python/pymdown-extensions/pymdown-extensions-6.3.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1
+
+DESCRIPTION="Extensions for Python Markdown."
+HOMEPAGE="
+	https://github.com/facelessuser/pymdown-extensions
+	https://pypi.org/project/pymdown-extensions
+"
+SRC_URI="https://github.com/facelessuser/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Docs require pymdownx.tabbed which is in master on github atm
+# but not in the 6.3.0 release tarballs for some reason
```

Comment:
As the comment here says, docs fail to build cause of a upstream bug that is already fixed in their master branch but has not been released yet. Perhaps we should leave the `doc` things commented out until the next release, otherwise emerge will fail if `USE="doc"`